web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
@mandawilson so how do you turn the session service on? if the url is defined in properties file, it's turned on by default? if that's not the case, we don't need to comment out the url in portal.properties.EXAMPLE file.